client/src/commands/stopFuelCore.ts

```diff
@@ -0,0 +1,11 @@
+import { exec } from 'child_process';
+import { window } from 'vscode';
+import { Config } from '../config';
+import updateFuelCoreStatus from '../status_bar/fuelCoreStatus';
+
+export default function stopFuelCore() {
+  exec(`pkill -9 fuel-core`, () => {
```

Comment:
This command isn't going to work on Windows, but it works on Mac and Linux

Reply:
Is lack of portability an issue? How would we even make this portable?

Reply:
We could have a bash script that checks for the existence of `pkill` , if it's not there looks for `taskkill` (Windows), and if that's not there, displays an error. I haven't tested it on Windows but I think in it's current form it would show an error popup with a message like "pkill not found". 

I'll open a separate issue for making this portable to Windows.

Reply:
https://github.com/FuelLabs/sway-vscode-plugin/issues/90

Reply:
Sounds good w.r.t. portability. How about

> Should we use SIGTERM (15) instead of SIGKILL (9), to allow fuel-core to shut down gracefully?

?